### ISSUE TITLE
RadhydroPulseGrey causes stochastic test timeouts and merge failure

### ISSUE THREAD

**Issue by BenWibking** [quokka-astro/quokka#624]
**Describe the bug**
Due to performance variability of the virtual machines that run GitHub actions, the RadhydroPulseGrey test stochastically times out when merging PRs. This causes the merge to fail and the PR to be ejected from the merge queue.

The problem parameters need to be modified so it doesn't take as long to run. The test suite already takes about 1hr 45min to run, even when it doesn't time out.

**To Reproduce**
1. Merge a PR
2. See error (e.g.: https://github.com/quokka-astro/quokka/actions/runs/8905036143/job/24455126405):
```
97% tests passed, 1 tests failed out of 34
Errors while running CTest

Total Test time (real) = 3890.12 sec

The following tests FAILED:
	 28 - RadhydroPulseGrey (Timeout)
```

**Additional context**
N/A

**Comment by BenWibking** [comment]
@chongchonghe This is preventing https://github.com/quokka-astro/quokka/pull/621 from merging.

**Comment by chongchonghe** [comment]
OK, I'll take on this.

**Comment by chongchonghe** [comment]
Addressed in #625
